docs/scripting.md

```diff
@@ -592,7 +592,7 @@ In addition to a regular expression and callback, the `hear` and `respond` funct
 
 The most important and most common metadata key is `id`. Every Listener should be given a unique name (options.id; defaults to `null`). Names should be scoped by module (e.g. 'my-module.my-listener'). These names allow other scripts to directly address individual listeners and extend them with additional functionality like authorization and rate limiting.
```

Comment:
Yeah, I can do that in a bit.

Thanks for all this hard work! It will make my hubot's maintenance muuuuuuuch better!